specification/liftrqumulo/resource-manager/Qumulo.Storage/preview/2022-10-12-preview/examples/FileSystems_CreateOrUpdate_MinimumSet_Gen.json

```diff
@@ -15,7 +15,7 @@
         },
         "provisioningState": "Accepted",
         "userDetails": {
-          "email": "aaaaaaaaaaaaaaaaaaaaaaa"
+          "email": "viptslwulnpaupfljvnjeq"
         },
         "delegatedSubnetId": "aaaaaaaaaa",
         "adminPassword": "ekceujoecaashtjlsgcymnrdozk",
```

Comment:
I wonder if we could use something less meaningful https://github.com/Azure/azure-rest-api-specs/blob/73e75c8d031d9d934d2ea2e431cf9dbeb3902195/specification/scvmm/ScVmm.Management/examples/2023-10-07/GuestAgents_Create_MaximumSet_Gen.json#L11 to avoid credential scan error in the sdk side.

Reply:
sure

Reply:
sure, thanks for pointing this out.